core/execution/engine.go

```diff
@@ -369,8 +369,8 @@ func (e *Engine) IsEligibleForProposerBonus(marketID string, value *num.Uint) bo
 }
 
 // SubmitMarket submits a new market configuration to the network.
-func (e *Engine) SubmitMarket(ctx context.Context, marketConfig *types.Market, proposer string) error {
-	return e.submitOrRestoreMarket(ctx, marketConfig, proposer, true)
+func (e *Engine) SubmitMarket(ctx context.Context, marketConfig *types.Market, proposer string, oos time.Time) error {
```

Comment:
`oos` is opening auction start? Shouldn't it be `oas` then? Or maybe something more obvious like `auctionStart` or whatever?